app/src/main/java/app/service/servicerequest/ServiceRequestService.java

```diff
@@ -178,15 +181,11 @@ private boolean requestAttributesHasAllRequiredServiceDefinitionAttributes(List<
         return requestCodes.containsAll(requiredCodes);
     }
 
-    private List<ServiceDefinitionAttributeDTO> buildUserResponseAttributesFromRequest(HttpRequest<?> request, List<ServiceDefinitionAttribute> serviceDefinitionAttributes) {
-
-        Argument<Map<String, String>> type = Argument.mapOf(String.class, String.class);
-        Optional<Map<String, String>> body = request.getBody(type);
```

Comment:
@HootDunk After migration, I found that `body` was null. I suspect it is due to the following breaking change outlined in https://docs.micronaut.io/latest/guide/#_body_annotation_on_controller_parameters . 

I resorted to deserializing the request body to a Map so that we have access to the `attribute[attrValue]` form keys.